### DISCUSSION TITLE
Obtain Row Reduced Echelon Form

### DISCUSSION THREAD

**Discussion by cdrnet** [mathnet/mathnet-numerics#807] [Ideas]
From: https://mathnetnumerics.codeplex.com/workitem/5689

> A way to obtain row echelon form from Gaussian Elimination for the Matrices. And Reduced Row Echelon form from Guass-Jordan elimination.

**Comment by jiedxu** [comment]
It seems to be implemented in [Accord.NET Framework](https://github.com/accord-net/framework/blob/master/Sources/Accord.Math/Matrix/ReducedRowEchelonForm.cs), though I haven't checked it.
